### ISSUE TITLE
libs don't have `to_s`

### ISSUE THREAD

**Issue by asterite** [crystal-lang/crystal#8867]
```crystal
require "spec"

lib LibFoo
end

describe LibFoo do
end
```

Fails with:

```
In /usr/local/Cellar/crystal/0.33.0/src/spec/methods.cr:18:44

 18 | Spec.root_context.describe(description.to_s, file, line, end_line, focus, tags, &block)
                                             ^---
Error: undefined fun 'to_s' for LibFoo
```

But for other types it works fine. This should be relatively easy to fix.

**Comment by asterite** [comment]
Actually, maybe a bug, but it would be confusing because lib functions are called that way. Closing.

**Comment by straight-shoota** [comment]
I think it would make perfectly sense to have this method. Being able to call `#to_s` on any object is quite a fundamental concept. And a lib type might not be an actual object, but it quacks like one. `to_s` could be an invalid function name to avoid overriding it.

Alternatively, we could consider handling this specific case in `spec` differently. Not sure how that could be done, though. Is there a way to treat `description` differently, when it's a lib type? I suppose the meta model is a bit sketchy in that regard.

**Comment by nobodywasishere** [comment]
I also agree there should be a `#to_s` method for libs, I ran into this issue today while writing specs. The only issue I'd see is if someone's trying to bind a method with the same name from a C library, which would no longer be possible, but not sure how likely that'd be to encounter.
